docs/UsersGuide/source/QuickStart.rst

```diff
@@ -0,0 +1,118 @@
+.. _QuickStart:
+
+**********************
+HAFS Quick Start Guide
+**********************
+
+Currently, the HAFS application works on these NOAA HPC platforms: wcoss_dell_p3, wcoss_cray, hera, jet, orion.
+
+=================================
+Clone and Checkout the Repository
+=================================
+
+.. code-block:: console
+
+    git clone <-b BRANCH> --recursive https://github.com/hafs-community/HAFS.git
+
+Select the branch to clone by setting the ``<-b BRANCH>`` option to the branch name.
+
+.. note::
+   ``develop`` is the default branch.
+
+======================
+Build and Install HAFS
+======================
+
+.. code-block:: console
+
+    cd HAFS/sorc
+    ./install_hafs.sh > install_hafs.log 2>&1
+
+.. Hint::
+   Got errors? Look into the ``HAFS/sorc/logs`` directory.
+
+------------------------
+Parts of install_hafs.sh
+------------------------
+
+* ``machine-setup.sh`` Determine shell, Identify machine, and Load modules
+
+* ``build_all.sh`` Compile components: forecast, post, tracker, utils, tools, hycom, ww3, and gsi
+
+* ``install_all.sh`` Copy executables to exec directory
+
+* ``link_fix.sh`` Link fix files (fix files are available on disk)
+
+===================
+Run the HAFS System
+===================
+
+----------------
+Edit system.conf
+----------------
+
+.. code-block:: console
+
+    cd HAFS/parm
+    cp system.conf.<system> system.conf
+    vi system.conf
```

Comment:
It may make sense just to instruct users to open the file in the command line with their preferred editor and note that `vi system.conf` is one option. That will make things more flexible, since there's vim, MacPorts, and a host of other tools people use.